phpseclib/Common/Functions/Objects.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/**
+ * Common Object Functions
+ *
+ * PHP version 5
+ *
+ * @category  Common
+ * @package   Functions\Objects
+ * @author    Jim Wigginton <terrafrost@php.net>
+ * @copyright 2016 Jim Wigginton
+ * @license   http://www.opensource.org/licenses/mit-license.html  MIT License
+ * @link      http://phpseclib.sourceforge.net
+ */
+
+namespace phpseclib\Common\Functions;
+
+/**
+ * Common Object Functions
+ *
+ * @package Functions\Objects
+ * @author  Jim Wigginton <terrafrost@php.net>
+ */
+abstract class Objects
```

Comment:
RSA.php has been significantly redesigned for the master branch. At this point, probably any change I make to RSA.php in the 1.0 / 2.0 branches will result in a merge conflict in the master branch. That is how significant the changes to RSA.php are.

SSH2.php is due to get some significant changes too but it has not had them. Right now, SSH2.php, as it currently exists in master, is pretty much 2.0's SSH2.php, which, in turn, is pretty much 1.0's SSH2.php, just namespaced. ie. a change made 1.0 will probably /not/ result in a merge conflict in the master branch's SSH2.php.

And this visibility changing stuff is used primarily by SSH2.php.

There are several alternatives I could be doing.

1. I could change the visibility only after every /big/ change I have planned has been made.
2. I could change the visibility to the files as big changes are being made to them. If I did that then RSA.php would have the visibility set but SSH2.php would not.
3. I could just delete the Objects class, retain the visibility changes, let the unit tests fail and just not care because the master branch isn't production ready anyway.

The option I've somewhat arbitrarily chosen is to update the visibility all at once and use hackish approach to get the unit tests working for the time being until I have had a chance to more comprehensively revamp those files.